nexus/intelligences/api/views.py

```diff
@@ -236,3 +248,43 @@ def destroy(self, request):
         return Response(
             status=status.HTTP_204_NO_CONTENT
         )
+
+
+class ContentBaseFileViewset(ModelViewSet):
```

Comment:
Change ModelViewset to retrive, create and list mixin. Or add usecases for the other mixins as well to use model mixin.